pyweather.py

```diff
@@ -1342,6 +1365,30 @@ def radar_clearImages():
     input()
     sys.exit()
 
+if showyesterdayonsummary == True:

```

Comment:
You'll want to add a conditional, that will run this code if prefetching yesterday's weather is true.

Reply:
Done